### PR TITLE
DEW IT

### PR DESCRIPTION
**Pull request by Blackstorm-21758**
You can't resist the powers of the DANK SIDE

### CODE REVIEW

File: aliases.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def aliases(hero):
 		hero='Gazlowe'
 	elif hero in ['graymane','gm']:
 		hero='Greymane'
-	elif hero in ['best hero']:
+	elif hero in ['genji','best hero','mada mada','virgin weaboo']:
```
Comment (Asddsa76) [inline aliases.py]: remove "virgin"

File: aliases.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ def aliases(hero):
 		hero='Johanna'
 	elif hero in ['jr','junk']:
 		hero='Junkrat'
-	elif hero in ['kt','kael','kaelthas']:#He came to HotS first. Use KTZ for your beloved lich.
+	elif hero in ['kt','kael','kaelthas']:# "He came to HotS first. Use KTZ for your beloved lich." <-- words of a virgin weaboo
```
Comment (Asddsa76) [inline aliases.py:37]: >:(

File: aliases.py [hunk 3]
```diff
@@ -107,4 +107,38 @@ def abilityAliases(name):
 		name='press'
 	elif name in ['mcs']:
 		name='merciless'
-	return name
\ No newline at end of file
+	elif name in ['wotw']:
+		name='way of the wind'
+	elif name in ['woi']
```
Comment (Asddsa76) [inline aliases.py]: You need colons at the end of each of these lines that start with elif

**Review by Asddsa76 — CHANGES_REQUESTED** [review]

File: aliases.py (hunk 1 above)
Comment (Blackstorm-21758) [reply aliases.py]: how about no

File: aliases.py (hunk 2 above)
Comment (Blackstorm-21758) [reply aliases.py:37]: 0:)

File: aliases.py (hunk 3 above)
Comment (Blackstorm-21758) [reply aliases.py]: no u

**Review by Asddsa76 — APPROVED** [review]

**Pull request merged**